pkgs/by-name/py/pywalfox-native/package.nix

```
@@ -10,6 +10,21 @@ python3.pkgs.buildPythonApplication rec {
     hash = "sha256-Wec9fic4lXT7gBY04D2EcfCb/gYoZcrYA/aMRWaA7WY=";
   };
 
+  postInstall = ''
+    # Overwrite the original wrapper script with a new one that has the
+    # appropriate executable paths.
+    cat > $out/lib/${python3.libPrefix}/site-packages/pywalfox/bin/main.sh <<'EOF'
+    #!${stdenv.shell}
+      ${placeholder "out"}/bin/pywalfox start
```

```suggestion
      $out/bin/pywalfox start
```

It may be better if you override this script with `makeShellWrapper` directly.

---

@Aleksanaa Since the original `main.sh` directly calls `python pywalfox` (https://github.com/Frewacom/pywalfox-native/blob/master/pywalfox/bin/main.sh#L10) I'd need to write something along these lines

```nix
  postInstall = ''
    export PYWALFOX=$out/lib/${python3.libPrefix}/site-packages/pywalfox/

    chmod +x $PYWALFOX/bin/main.sh

    makeShellWrapper $PYWALFOX/bin/main.sh $out/bin/patched-main.sh \
      --prefix PATH : ${lib.makeBinPath [ (python3.withPackages [ self??? ]) ]}

    substituteInPlace $PYWALFOX/config.py \
      --replace "BIN_PATH_UNIX = os.path.join(APP_PATH, 'bin/main.sh')" \
      "BIN_PATH_UNIX = \"$out/bin/patched-main.sh\""

	# ....
  '';
```

which felt a bit overcomplicated and I am still not entirely sure how would I "correctly" handle the `self` reference. Hence why I opted to just replace the original script with a new one.